wasm_samples/executor_remove_permission/src/lib.rs

```diff
@@ -15,17 +15,18 @@ static ALLOC: GlobalDlmalloc = GlobalDlmalloc;
 
 getrandom::register_custom_getrandom!(iroha_executor::stub_getrandom);
 
-#[derive(Constructor, ValidateEntrypoints, Validate, Visit)]
+#[derive(Constructor, Visit, Execute, Entrypoints)]
```

Comment:
IMO carries too much mental overhead for the end user. Can we simplify it?

Reply:
If they always used together we can combine them.

Reply:
#5106